src/lmptype.h

```diff
@@ -200,6 +203,23 @@ typedef int bigint;
 #define _noalias
 #endif
 
+// declaration to turn off optimization for specific functions
+// and avoid compiler warnings about variable tracking
+
+#if defined(__clang__)
+#  define _noopt __attribute__((optnone))
+#elif defined(__INTEL_COMPILER)
```

Comment:
@akohlmey It looks like the Intel compiler doesn't support `attributes`; instead of attributes, one can also use:

```
#pragma GCC push_options
#pragma GCC optimize ("-O2")
void foo()
{
   ...
}
#pragma GCC pop_options
```

And many more compilers like the Intel compiler and clang support something similar to this as well, see https://stackoverflow.com/questions/31373885/how-to-change-optimization-level-of-one-function.

Reply:
But probably not worth the trouble...